### PR TITLE
Noop driver for scale testing runner

### PR DESCRIPTION
**Pull request by shubham149**
# Commit Checklist

Thank you for creating a pull request! To help us review / merge this can you make sure that your PR adheres as much as possible to the following.

## The Basics

If you are adding new functionality, please provide evidence of the new functionality.

### CODE REVIEW

File: internal/drivers/noop/option.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package noop
+
+type Option func(*config)
+
+// WithRootDirectory returns an OS specific temp directory
+func WithRootDirectory() Option {
+	return func(p *config) {
+		p.rootDir = "/tmp/noop"
```
Comment (vistaarjuneja) [inline internal/drivers/noop/option.go:8]: should this be configurable?

File: internal/drivers/noop/driver.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	leIP              string
+}
+
+func New(opts ...Option) (drivers.Driver, error) {
+	p := new(config)
+	for _, opt := range opts {
+		opt(p)
+	}
+
+	p.hibernateWaitSecs = 5
```
Comment (vistaarjuneja) [inline internal/drivers/noop/driver.go:30]: could be made configurable (from yaml) with default values? feel free to ignore if you think it's unnecessary

File: internal/drivers/manager.go [hunk 3]
```diff
@@ -558,6 +558,7 @@ func (m *Manager) StartInstance(ctx context.Context, poolName, instanceID string
 		return nil, fmt.Errorf("start_instance: pool name %q not found", poolName)
 	}
 
+	logrus.WithField("instanceID", instanceID).Infoln("Starting vm from hibernate state")
```
Comment (vistaarjuneja) [inline internal/drivers/manager.go:561]: VM might not be hibernated here, we should move this log message after the hibernate check

File: internal/drivers/noop/option.go (hunk 1 above)
Comment (shubham149) [reply internal/drivers/noop/option.go:8]: not required

File: internal/drivers/noop/driver.go (hunk 2 above)
Comment (shubham149) [reply internal/drivers/noop/driver.go:30]: not required for now

**Review by vistaarjuneja — APPROVED** [review]

**Pull request merged**